src/FirmwarePlugin/PX4/PX4FirmwarePlugin.cc

```diff
@@ -174,7 +174,7 @@ bool PX4FirmwarePlugin::setFlightMode(const QString& flightMode, uint8_t* base_m
 
 int PX4FirmwarePlugin::manualControlReservedButtonCount(void)
 {
-    return 8;   // 8 buttons reserved for rc switch simulation
+    return 0;   // 0 buttons reserved for rc switch simulation
```

Comment:
That's true but COM_RC_IN_MODE=2 is not a viable flight mode in my experience. IIRC that disable all RC checks which is a no-go for actual flying vehicles. 

I agree that I've changed the semantics of what it means; ultimately that is Lorenz's call but in my experience having to deal with treating momentary buttons like position toggle switches was a royal pain and prone to problems. In testing I found that older version of PX4 firmware were only marginally comfortable or trustworthy to fly; there were certain mode transitions that were not possible atomically because the user had to press a button to toggle one mode before the other.

I suppose we could still send the reserved button bit states for older PX4 firmware revisions that do not support the complete atomic mode switch behavior. The point of confusion there will be both conflicting sources of mode switch (decoded button press -> switch position vs. bound button SET_MODE commands), and older users wondering why all the auto modes do not properly transition if they have them bound.